### PR TITLE
container to create new quiz template

### PR DESCRIPTION
**Pull request by megsachdev**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/BioMediaLab/whiteboard/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?
PR adds the functionality to create new quiz template

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Currently, quiz template can be created only from backend

Issue Number: #28 

## What is the new behavior?
UI to create new quiz template

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/components/Description/Description.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
```
Comment (cliffpyles) [inline src/components/Description/Description.js]: Is it possible to rename this to something a little more appropriate? If we can't, we might want to reconsider if it's duplicate code by coincidence or by intent. As these form fields grow over the next few weeks, I could see that they would no longer be duplicates. Here is an article from Robert C. Martin about duplication by accident vs duplication by intent. http://www.informit.com/articles/article.aspx?p=1313447

File: src/containers/QuizTemplates/QuizTemplateCreate.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            graphqlOperation(createQuizTemplate, {
+                input:quiztemplate
+            })
+        )
+    }
+
+    const handleSubmit = () => {
+        saveQuizTemplate()
+            .then(data => {
+                dispatch({ type: 'RESET_QUIZ' })
```
Comment (cliffpyles) [inline src/containers/QuizTemplates/QuizTemplateCreate.js]: The naming conventions should match the ubiquitous language. Can we rename this to `RESET_QUIZ_TEMPLATE`?

File: src/components/Description/Description.js (hunk 1 above)
Comment (megsachdev) [reply src/components/Description/Description.js]: Done. Changed to PageHeader

File: src/containers/QuizTemplates/QuizTemplateCreate.js (hunk 2 above)
Comment (megsachdev) [reply src/containers/QuizTemplates/QuizTemplateCreate.js]: Done

Comment (cliffpyles) [reply src/containers/QuizTemplates/QuizTemplateCreate.js]: I'm still showing `RESET_QUIZ`. Did you push the changes?

Comment (cliffpyles) [reply src/containers/QuizTemplates/QuizTemplateCreate.js]: I see it now. I had the commit hash appended to the URL.

**Pull request merged**